js/script.js

```diff
@@ -17,3 +16,152 @@ navMenu.forEach((links) => links.addEventListener('click', () => {
+    sourceLink: '#',
+  },
+  {
+    name: 'Profesional Art Printing Data More',
+    image: 'assets/mobile/Portfolio2.svg',
+    imageDescription: 'Image Design',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    image: 'assets/mobile/Portfolio3.svg',
+    imageDescription: 'Portfolio image',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Website Portfolio',
+    image: 'assets/mobile/Portfolio3.svg',
+    imageDescription: 'Portfolio image placeholder',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+const modal = document.querySelector('.modal');
+const modalClose = document.querySelector('.close-button');
+
+modalClose.addEventListener('click', () => {
+  modal.classList.remove('modal-open');
+});
+
+function ShowModal(index) {
+  modal.classList.toggle('modal-open');
+  const modalTitle = document.querySelector('.name');
+  const modalImage = document.querySelector('.modal-image');
+  const projectDescription = document.querySelector('.project-description');
+  const techList = document.querySelector('.tech-container');
+  const seeLive = document.querySelector('.see-live');
+  const seeSource = document.querySelector('.see-source');
+  const project = projectsLists[index];
+
+  modalTitle.textContent = project.name;
+  modalImage.setAttribute('src', project.image);
+  projectDescription.textContent = project.description;
+  seeLive.setAttribute('href', project.link);
+  seeSource.setAttribute('href', project.source);
+
+  techList.innerHTML = '';
+
+  project.techList.forEach((tech) => {
+    const li = document.createElement('li');
+    li.innerText = tech;
+    techList.appendChild(li);
+  });
+}
+
+const seeProj1 = document.querySelector('.see-proj1');
+const seeProj2 = document.querySelector('.see-proj2');
+const seeProj3 = document.querySelector('.see-proj3');
+const seeProj4 = document.querySelector('.see-proj4');
+const seeProj5 = document.querySelector('.see-proj5');
+const seeProj6 = document.querySelector('.see-proj6');
+const seeProj7 = document.querySelector('.see-proj7');
+
+seeProj1.addEventListener('click', () => {
+  ShowModal(0);
+});
+
+seeProj2.addEventListener('click', () => {
+  ShowModal(1);
+});
+
+seeProj3.addEventListener('click', () => {
+  ShowModal(2);
+});
+
+seeProj4.addEventListener('click', () => {
+  ShowModal(3);
+});
+
+seeProj5.addEventListener('click', () => {
```

Comment:
- [optional] In order to save computational time and make your codebase clean, I think it would be better to assign a common classname for all the `see-projects` buttons, use `document.querySelectorAll('.projectclassname')`, loop through it (perhaps forEach) and add eventListener to the current element in iteration. ℹ️ 
```suggestion
const seeProj = document.querySelectorAll('.see-proj');

seeProj.forEach ((projectButton) =>
{
projectButton.addEventListener('click', () => {
  ShowModal();
});
})
```